### DISCUSSION TITLE
Delete a file in nnn to macOS Desktop Trash

### DISCUSSION THREAD

**Discussion by slferris** [jarun/nnn#1779] [Q&A]
How can I delete a file using nnn with macOS Sonoma and zsh by moving it to Desktop Trash? I have trash_cli installed with dependencies, but not sure how to proceed from there.

**Comment by slferris** [comment]
Correction: trash-cli

**Reply by jarun** [reply]
export NNN_TRASH=1

Please refer to: https://github.com/jarun/nnn/wiki/Usage#configuration
